src/main/haskell/kore/src/Kore/Attribute/Parser.hs

```diff
@@ -84,7 +99,10 @@ class Default atts => ParseAttributes atts where
     attributesParser :: Parser atts
 
 -- | One occurrence of an attribute, represented as a list of its arguments.
-newtype Occurrence = Occurrence { getOccurrence :: [CommonKorePattern] }
+data Occurrence = Occurrence
+    { sortParameters :: [Sort Object]
```

Comment:
All fields should be strict unless there is a good reason to do otherwise.

Reply:
Why are we not using the `StrictData` language setting?

Reply:
Because the style guide on which ours was based didn't use it, and I didn't know that we had other options. Not sure why the other people that contributed didn't suggest it.